src/httpserver/core.clj

```diff
@@ -11,8 +10,8 @@
 (defn -main [& args]
   (let [{:keys [options]} (arguments/build-map args)]
     (let [server (HttpServer. (:port options) (:directory options))
-          teapot-handler (teapot/new-teapot-handler)
-          ok-handler (ok/new-ok-handler)]
-      (.addRoute server "GET" "/coffee" teapot-handler)
-      (.addRoute server "GET" "/tea" ok-handler)
+          main-handler (handler/new-handler)
+          ]
+      (.addRoute server "GET" "/coffee" main-handler)
+      (.addRoute server "GET" "/tea" main-handler)
```

Comment:
Am I the only one sensing some duplication between this and the `cond` statement in the `handler-function-selector` namespace? I think this should be addressed pretty soon, but I'm fine with it being a separate PR. Just want to make sure you're thinking about it.